src/AccountLabel.jsx

```diff
@@ -1,52 +1,54 @@
 const React = require('react');
-const {bonds, toChecksumAddress} = require('oo7-parity');
-const {ReactiveComponent, Rimg} = require('oo7-react');
-const {isNullData} = require('oo7-parity');
-const {Label, Icon} = require('semantic-ui-react');
-const {AccountIcon} = require('./AccountIcon');
+const { bonds, toChecksumAddress, isNullData } = require('oo7-parity');
+const { ReactiveComponent, Rimg } = require('oo7-react');
+const { Label, Icon } = require('semantic-ui-react');
+const { AccountIcon } = require('./AccountIcon');
 
 class AccountLabel extends ReactiveComponent {
 	constructor () {
 		super(['address']);
 	}
 	render () {
 		if (this.state.address === null) {
```

Comment:
Can this.state.address be `undefined`? If this case (and in general) if would be better to use `if (!this.state.address)`.

Only use === null when you are explictly testing that it's ===null and not ===undefined

Reply:
yup updated